languages/thingtalk/load-thingpedia.js

```diff
@@ -590,8 +590,8 @@ class ThingpediaLoader {
             'query',
             { p_name: Type.String },
             new Ast.Table.Filter(null, table, namefilter, table.schema),
-            [`\${p_name}`],
-            [`\${p_name}`],
+            [`\${p_name}`, ...canonical.map((c) => `\${p_name} ${c}`)],
+            [`\${p_name}`, ...canonical.map((c) => `\${p_name} ${c}`)],
```

Comment:
Strictly speaking, this is English only, we should use langPack and let other languages override this.

Reply:
where is langPack?

Reply:
`this._langPack` in load-thingpedia is a reference to a file in lib/i18n
Anyway I didn't mean this as a real suggestion for this PR, more of mentioning it.
(Now that I think about it, you could also get the same effect as a standard Genie template, using `constant_name` and `base_noun_phrase`. Anyway not worth the effort.)